R/bcea.default.R

```diff
@@ -1,106 +1,124 @@
-  ##    and c so if ref=1 the first column is assumed to be associated with the reference intervention. 
-  ##    Intervention 1 is assumed the default reference. All others are considered comparators.
-  ## 3. A string vector "interventions" including the names of the interventions. If none is provided 
-  ##    then labels each as "intervention1",...,"interventionN".
-  ## 4. The value Kmax which represents the maximum value for the willingness to pay parameter. If none 
-  ##    is provided, then it is assumed Kmax=50000.
-  ## 5. A(n optional) vector wtp including the values of the willingness to pay grid. If not specified
-  ##    then BCEA will construct a grid of 501 values from 0 to Kmax. This option is useful when 
-  ##    performing intensive computations (eg for the EVPPI)
-  ##
-  ## OUTPUTS:
-  ## Graphs & computed values for CE Plane, ICER, EIB, CEAC, EVPI 
+
+#' Default function
+#'
+#' Compute a Bayesian cost-effectiveness analysis of two or more interventions
+#'
+#' INPUTS:
+#' 1. Two objects (e,c). These can be directly computed in a simulation object "sim" from JAGS/BUGS, 
+#'    or derived by postprocessing of "sim" in R. The objects (e,c) have dimension (n.sim x number of 
+#'    interventions) and contain n.sim simulated values for the measures of effectiveness and costs 
+#'    for each intervention being compared. 
+#' 2. The reference intervention as a numeric value. Each intervention is a column in the matrices e 
+#'    and c so if ref=1 the first column is assumed to be associated with the reference intervention. 
+#'    Intervention 1 is assumed the default reference. All others are considered comparators.
+#' 3. A string vector "interventions" including the names of the interventions. If none is provided 
+#'    then labels each as "intervention1",...,"interventionN".
+#' 4. The value Kmax which represents the maximum value for the willingness to pay parameter. If none 
+#'    is provided, then it is assumed Kmax=50000.
+#' 5. A(n optional) vector wtp including the values of the willingness to pay grid. If not specified
+#'    then BCEA will construct a grid of 501 values from 0 to Kmax. This option is useful when 
+#'    performing intensive computations (eg for the EVPPI)
+#'
+#' @param e 
+#' @param c 
+#' @param ref 
+#' @param interventions 
+#' @param Kmax 
+#' @param wtp 
+#' @param plot 
+#'
+#' @return Graphs & computed values for CE Plane, ICER, EIB, CEAC, EVPI 
+#' @export
+#'
+#' @examples
+bcea.default <- function(e,
+                         c,
+                         ref = 1,
+                         interventions = NULL,
+                         Kmax = 50000,
+                         wtp = NULL,
+                         plot = FALSE) {
   
   # Set the working directory to wherever the user is working, if not externally set
-  if(!exists("working.dir")){working.dir <- getwd()}
+  if(!exists("working.dir")){working.dir <- here::here()}
```

Comment:
I'd suggest removing this line as there are no other references in the code to `working.dir` (no other references to the object throughout the code), or adding "here" to imported packages as the line would fail otherwise